crt_portal/cts_forms/templates/forms/confirmation.html

```diff
@@ -2,16 +2,22 @@
 {% load i18n %}
 {% load static %}
 
-{% block main_class %}class="confirmation-page"{% endblock %}
+{% block body_class %}class="confirmation-page"{% endblock %}
 
 {% block page_header %}
 <header class="crt-form-header">
   <div class="grid-container">
     <div class="grid-row grid-gap flex-child">
       <div class="tablet:grid-col-9 tablet:grid-offset-1">
+        <div class="doj-brand-print">
+          <img src="{% static "img/doj-logo-footer.png" %}" alt="" height="38" class="margin-right-105" />
+          <span>{% trans "United States Department of Justice" %}</span>
+          <span>{% trans "Civil Rights Division" %}</span>
+          <span class="doj-brand-url">www.civilrights.justice.gov</span>
```

Comment:
Let's trim the `www` to keep this consistent with our social media images

```suggestion
          <span class="doj-brand-url">civilrights.justice.gov</span>
```